<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>400</width><height>300</height></rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация — Магазин обуви</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin"><number>40</number></property>
   <property name="topMargin"><number>20</number></property>
   <property name="rightMargin"><number>40</number></property>
   <property name="bottomMargin"><number>20</number></property>
   <item>
    <widget class="QLabel" name="logo_label">
     <property name="text"><string/></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
     <property name="maximumSize"><size><width>16777215</width><height>60</height></size></property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text"><string>Вход в систему</string></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
     <property name="font">
      <font><pointsize>16</pointsize><bold>true</bold></font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text"><string>Email:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="email_input">
       <property name="placeholderText"><string>Введите email</string></property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text"><string>Пароль:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password_input">
       <property name="placeholderText"><string>Введите пароль</string></property>
       <property name="echoMode"><enum>QLineEdit::Password</enum></property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>8</height></size></property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="login_btn">
     <property name="text"><string>Войти</string></property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="guest_btn">
     <property name="text"><string>Войти как гость</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
